src/state/apis/zapper/zapperApi.ts

```diff
@@ -131,18 +132,18 @@ export const zapperApi = createApi({
 
             const name = underlyingAssets.every(asset => asset && asset.symbol)
               ? `${underlyingAssets.map(asset => asset?.symbol).join('/')} Pool`
-              : appTokenData.displayProps.label.replace('WETH', 'ETH')
+              : (appTokenData.displayProps?.label ?? '').replace('WETH', 'ETH')
             const icons = underlyingAssets.map((underlyingAsset, i) => {
-              return underlyingAsset?.icon ?? appTokenData.displayProps.images[i]
+              return underlyingAsset?.icon ?? appTokenData.displayProps?.images[i] ?? ''
             })
 
             acc.byId[assetId] = makeAsset({
               assetId,
-              symbol: appTokenData.symbol,
+              symbol: appTokenData.symbol ?? '',
               // WETH should be displayed as ETH in the UI due to the way UNI-V2 works
               // ETH is used for depositing/withdrawing, but WETH is used under the hood
               name,
-              precision: appTokenData.decimals,
+              precision: appTokenData.decimals ?? 0,
```

Comment:
if we don't have precision log and skip the asset, we never want to guess on this

Reply:
9c3049f80a0c87e4de5094e359e857d412a66e11